src/index.ts

```diff
@@ -8,26 +8,39 @@ import { Scenario } from './model/Scenario';
 import { TestResult } from './model/TestResult';
 import { loadAllScenarios, loadScenariosById } from './scenarioLoading';
 import { loadYamlConfiguration } from './yamlParsing';
+import { stringify } from 'querystring';
 
 const RESULTS: Map<string, TestResult[]> = new Map();
 
-const NUM_PARALLEL_RUNS = 10;
-
 let OUT_DIR = '';
 
 // increase the pixel-size (width/height) limit of PlantUML (the default is 4096 which is not enough for some diagrams)
 process.env.PLANTUML_LIMIT_SIZE = '16384';
 
-export const runScenario = (
-    scenarioPath: string,
+interface RunConfiguration {
+    numberOfScenariosRunInParallel?: number;
+    environmentNameToBeUsed?: string;
+}
+
+export const runMultipleSceanriosWithConfig = (
     actionDir: string,
     outDir = 'out',
-    envConfigFile: string,
+    envConfigDir: string,
+    runConfig: RunConfiguration,
+    ...scenarioPaths: Array<string>
```

Comment:
The linter is configured to prefer `T[]` over `Array<T>`:
"Array type using 'Array<string>' is forbidden. Use 'string[]' instead.eslint(@typescript-eslint/array-type)"

So please use String[] instead.

Reply:
ok